recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -29,16 +33,24 @@ def _min_cppstd(self):
     def _compilers_minimum_version(self):
         return {"gcc": "11",
                 "Visual Studio": "16.9",
+                "msvc": "1928",
                 "clang": "13",
-                "apple-clang": "13",
-                }
+                "apple-clang": "13"}
+
+    def configure(self):
+        version = Version(self.version.strip("v"))
+        if version.major < 2022 or (version.major == 2022 and version.minor < 9):
+            self.license = "MIT"
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.build.check_min_cppstd(self, self._min_cppstd)
-        if self.settings.compiler == "Visual Studio":
+        if self.info.settings.get_safe("compiler.cppstd"):
```

Comment:
```suggestion
        # FIXME: Need to use self.info.settings for header-only
        if self.settings.get_safe("compiler.cppstd"):
```
So far, we can not use `self.info.settings` as compiler is empty and the CI is not able to detect the minimal compiler version.